plugins/inputs/cpufreq/README.md

```diff
@@ -0,0 +1,29 @@
+# CPUFreq Input Plugin For Telegraf Agent
+
+The CPUFreq plugin collects the current CPU's frequency. This plugin work only with linux.
+
+## Configuration
+
+```toml
+[[inputs.cpufreq]]
+  ## Path for sysfs filesystem.
+  ## See https://www.kernel.org/doc/Documentation/filesystems/sysfs.txt
+  ## Defaults:
+  # host_sys = "/sys"
+  ## Gather CPU throttles per core
+  ## Defaults:
+  # gather_throttles = false
+```
+
```

Comment:
Could you please add a _Metric_ section describing the fields similar to https://github.com/influxdata/telegraf/pull/8342!?